installer/install.iss

```diff
@@ -1708,29 +1719,37 @@ begin
 
     // 4th choice
     Top:=TopOfLabels;
+    CbbEditor.Items.Add('Use VSCodium as Git'+#39+'s default editor');
+    if (VSCodiumUserInstallation=False) then
+        CreateItemDescription(EditorPage,'<RED>(NEW!)</RED> <A HREF=https://vscodium.com///>VSCodium</A> provides Free/Libre Open Source Software Binaries of VSCode with the same features, but without telemetry/tracking of Microsoft or any non-floss code parts. It comes with built-in support for JavaScript,'+#13+'TypeScript and Node.js and has a rich ecosystem of extensions for other'+#13+'languages (such as C++, C#, Java, Python, PHP, Go) and runtimes (such as'+#13+'.NET and Unity).'+#13+#13+'Use this option to let Git use VSCodium as its default editor.',Top,Left,LblEditor[GE_VSCodium],False)
+    else
+        CreateItemDescription(EditorPage,'<RED>(NEW!)</RED> <A HREF=https://vscodium.com///>VSCodium</A> provides Free/Libre Open Source Software Binaries of VSCode with the same features, but without telemetry/tracking of Microsoft or any non-floss code parts. It comes with built-in support for JavaScript,'+#13+'TypeScript and Node.js and has a rich ecosystem of extensions for other'+#13+'languages (such as C++, C#, Java, Python, PHP, Go) and runtimes (such as'+#13+'.NET and Unity).'+#13+'<RED>(WARNING!) This will be installed only for this user.</RED>'+#13+#13+'Use this option to let Git use VSCodium as its default editor.',Top,Left,LblEditor[GE_VSCodium],False);
+
+    // 5th choice
+    Top:=TopOfLabels;
     CbbEditor.Items.Add('Use Visual Studio Code as Git'+#39+'s default editor');
     if (VisualStudioCodeUserInstallation=False) then
-        CreateItemDescription(EditorPage,'<RED>(NEW!)</RED> <A HREF=https://code.visualstudio.com//>Visual Studio Code</A> is an Open Source, lightweight and powerful editor'+#13+'running as a desktop application. It comes with built-in support for JavaScript,'+#13+'TypeScript and Node.js and has a rich ecosystem of extensions for other'+#13+'languages (such as C++, C#, Java, Python, PHP, Go) and runtimes (such as'+#13+'.NET and Unity).'+#13+#13+'Use this option to let Git use Visual Studio Code as its default editor.',Top,Left,LblEditor[GE_VisualStudioCode],False)
+        CreateItemDescription(EditorPage,'<RED>(NEW!)</RED> <A HREF=https://code.visualstudio.com//>Visual Studio Code</A> is a lightweight and powerful editor'+#13+'running as a desktop application with an open source core. It comes with built-in support for JavaScript,'+#13+'TypeScript and Node.js and has a rich ecosystem of extensions for other'+#13+'languages (such as C++, C#, Java, Python, PHP, Go) and runtimes (such as'+#13+'.NET and Unity).'+#13+#13+'Use this option to let Git use Visual Studio Code as its default editor.',Top,Left,LblEditor[GE_VisualStudioCode],False)
```

Comment:
Can you please explain why you think this specific change is contentious? Actually I've set up a new win32 box. To be able to work with it I've searched for the official Git installer which brought me to this page. VSCode sounded good, so I've clicked on the link, got the download and the first thing its installer told me that there's a difference between the "product" and the "software compiled from the code" [https://code.visualstudio.com/license]:

> This license applies to the **Visual Studio Code product**. _Source code_ for Visual Studio Code is available at https://github.com/Microsoft/vscode _under the MIT license_ agreement. The software is licensed, not sold. **This agreement only gives you some rights to use the software. Microsoft reserves all other rights.** Unless applicable law gives you more rights despite this limitation, you may use the software only as expressly permitted in this agreement. In doing so, you must comply with any technical limitations in the software that only allow you to use it in certain ways. **You may not**

* reverse engineer, decompile or disassemble the software, or otherwise attempt to derive the source code for the software except and solely to the extent required by third party licensing terms governing use of _certain open source components_ that _may be included in the software_; [...]
* **share, publish**, rent or lease the software, **or provide the software as a stand-alone offering for others to use**

This was the place where I've aborted the installation and after reading about this issue and the auto-telemetry/tracking I've wondered how to "just get the software for free use" and was happy to find VSCodium as a result.

... so after my personal story about how Git helped me to find VSCodium after actually reading the license to find out that VSCode _may_ be "open source" but definitely not free software (as the MIT license to its source actually does) - back to my question:

Why do you see this change contentious?

Reply:
I see it as contentious because it was slipped into a commit that claims to add VSCodium, yet it also modified a part that was not about VSCodium (but about VSCode instead).

Maintaining Git for Windows is already taking too much time, I cannot afford to get into any discussion about Free/Libre code on top of that.

Reply:
Thanks for answering.

> I see it as contentious because it was slipped into a commit that claims to add VSCodium, yet it also modified a part that was not about VSCodium (but about VSCode instead).

that would be bad, but actually the commit message explicit says so:

> and minor change in the text to VisualStudioCode to distinguish its MIT-licensed "core" with the provided un-free binaries (+telemetry/tracking) between the option added.

But it is OK for me to not argue about Free/Libre code.
Discussion resolved for both of us. Thanks for doing the work on this repo.